example-crates/example_electrum/src/main.rs

```
@@ -84,6 +84,7 @@ fn main() -> anyhow::Result<()> {
         Network::Testnet => "ssl://electrum.blockstream.info:60002",
         Network::Regtest => "tcp://localhost:60401",
         Network::Signet => "tcp://signet-electrumx.wakiyamap.dev:50001",
+        _ => todo!(),
```

In a219219691bb1c6d47e8f8a9df3100f92eee0baf:

`todo` is not a bad choice for this but I'd panic with "Unknown network" or something so it's clear this isn't a code todo.